.vscode/extensions.json

```diff
@@ -1,3 +1,8 @@
 {
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "antfu.pnpm-catalog-lens"]
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "antfu.pnpm-catalog-lens",
+    "unifiedjs.vscode-mdx"
```

Comment:
To see lint hints for MDX files in VSCode, please install the [MDX VSCode extension](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx).